applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -1204,11 +1284,14 @@ int le_audio_volume_mute(void)
 int le_audio_play_pause(void)
 {
 	int ret;
-
-	ret = ble_mcs_play_pause(NULL);
-	if (ret) {
-		LOG_WRN("Failed to change streaming state");
-		return ret;
+	if (IS_ENABLED(CONFIG_STREAM_BIDIRECTIONAL)) {
```

Comment:
Is the if and else mixed here? if BIDIR then it should print the "play pause not supported"